tests/parser/test_parser.py

```diff
@@ -390,9 +390,18 @@ def test_alternative_wildcard_wrap(self, wildcard_wrap: str, template: str):
         assert variant.values[1].literal == "B"
         assert variant.values[2].wildcard == "some/wildcard"
 
-    @pytest.mark.parametrize("immediate", (False, True))
-    def test_variable_commands(self, immediate: bool):
-        op = "=!" if immediate else "="
+    @pytest.mark.parametrize(
+        ("immediate", "preserve"),
+        [
+            (False, False),
+            (False, True),
+            (True, False),
+            (True, True),
+        ],
+    )
```

Comment:
This will have the same combinatorial effect, plus nicer test IDs:
```suggestion
    @pytest.mark.parametrize("immediate", (False, True), ids=("delayed", "immediate"))
    @pytest.mark.parametrize("preserve", (False, True), ids=("not-preserved", "preserved"))
```